static/js/vue-cdr-access/src/translations.js

```diff
@@ -41,7 +41,6 @@ export default {
             additional_metadata: "View Additional Metadata",
             available_date: "Available after {available_date}",
             collection: "Collection",
-            collection_id: "Archival Collection ID",
```

Comment:
I assume you're going to need this back for https://unclibrary.atlassian.net/browse/BXC-4463 ? Not that you have to add it back in this PR.